newjersey/navigator.business.nj.gov
# feat: [#186775240] Add health check to formation

## Description



### Ticket



[#186775240](https://www.pivotaltracker.com/story/show/)

### Approach



### Steps to Test


Run yarn start:dev and go to http://localhost:5002/local/health/webservice/formation-health-check
### Notes



## Code author checklist
